src/behaviour.rs

```diff
@@ -26,6 +26,8 @@ use libp2p::core::either::EitherOutput;
 use libp2p::core::{connection::ConnectionId, Multiaddr, PeerId};
 use libp2p::swarm::derive_prelude::{ConnectionClosed, DialFailure, FromSwarm, ListenFailure};
 #[cfg(feature = "compat")]
+use libp2p::swarm::SubstreamProtocol;
```

Comment:
please fold this into the following import

Reply:
Fixed